servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/DefaultLoadBalancer.java

```diff
@@ -582,4 +607,136 @@ public int hostSetSize() {
             return 0;
         }
     }
+
+    // TODO: weight and priority need representation on the ServiceDiscovererEvent.
+    private static double getWeight(ServiceDiscovererEvent<?> event) {
```

Comment:
nit: we are not really using bean-style prefixes in the code base. Not a big deal it just stands out a bit - maybe consider renaming the methods?